uxarray/grid/coordinates.py

```diff
@@ -235,3 +234,88 @@ def _populate_lonlat_coord(grid):
             "long_name": "latitude of mesh nodes",
             "units": "degrees_north",
         })
+
+
+def _populate_centroid_coord(self, repopulate=False):
```

Comment:
Missed this in my review! Please replace all instances of `self` with `grid` throughout this function.